app/views/tournaments/index.haml

```diff
@@ -0,0 +1,2 @@
+%section
+  %h1 Start a table soccer tournament
```

Comment:
Here's a fix for issues like these (the newline, not the adccidental addition):
https://github.com/railslove/kickerapp/pull/114